src/common/app.config.js

```diff
@@ -387,9 +387,9 @@ export const appConfig = /* @ngInject */ function (envServiceProvider, $compileP
     RADIO_STATION_SELECT_ERROR: 'There was an error selecting a radio station.',
     PREFERRED_RADIO_STATION: 'Preferred Radio Station: ',
     SOMETHING_WENT_WRONG: 'Something went wrong. Please try again',
-    OKTA_EMAIL_ALREADY_EXISTS: 'The email address you used belongs to an existing Okta user.',
-    OKTA_ERROR_WHILE_SAVING_EMAIL: 'There was an error saving your email address. Make sure it was entered correctly.',
-    OKTA_ERROR_WHILE_SAVING_DATA: 'There was an error saving your contact info. Please try again or contact eGift@cru.org for assistance.'
```

Comment:
We want to keep this. I know we're not currently using it, but if there was an error we need to show this text. I have created a task to create these pages.

Reply:
It prob needs to be added to the ES translation too.